docs/reference/commandline/swarm_join_token.md

```diff
@@ -65,6 +66,12 @@ SWMTKN-1-3pu6hszjas19xyp7ghgosyx9k8atbfcr8p2is99znpy26u2lkl-b30ljddcqhef9b9v4rs7
 ### `--rotate`
 
 Update the join token for a specified role with a new token and print the token.
+Invalidate the old token. Rotating a token does not effect existing nodes in the
```

Comment:
Rotating a join-token means that no new nodes will be able to join the swarm using the old token. Note that this rotation does not affect existing nodes in the swarm, since the join token is only used for authorizing new nodes to the cluster.